src/item.rb

```diff
@@ -0,0 +1,25 @@
+require 'date'
+
+class Item
+  attr_reader :id, :genre, :author, :source, :label, :publish_date, :archived
+
+  def initialize(genre, author, source, label, publish_date)
+    @id = rand(1..1000)
+    @genre = genre
+    @author = author
+    @source = source
+    @label = label
+    @publish_date = publish_date
+    @archived = false
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+  end
```

Comment:
- [ ] for the item class the "has one" association should be set

Reply:
Thank you @mikethreels , we will add the associations.